src/Telemetry/Events/Event_Subscriber.php

```diff
@@ -21,6 +21,8 @@
  */
 class Event_Subscriber extends Abstract_Subscriber {
 
+	private static $events = [];
```

Comment:
```suggestion
	/**
	 * @var array
	 */
	private static $events = [];
```

Reply:
I actually just pushed a doc for this :chuckle:

Reply:
Actually, maybe that version # should be `2.3.0.1`?